projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -46,6 +46,26 @@ public static function register() {
 		}
 	}
 
+	/**
+	 * Registers inline scripts which populate `window.JetpackTiledGalleryOptions` in Javascript.
+	 *   window.JetpackTiledGalleryOptions.isAtomicSite (Boolean)  - Is the site atomic?
+	 *   window.JetpackTiledGalleryOptions.isPrivateSite (Boolean) - Is the site private?
+	 *
+	 * @return void
+	 **/
+	public static function inject_javascript_options() {
+		$options = array(
+			// 'isAtomicSite'  => jetpack_is_atomic_site(),
+			// ################################### WARNING #####################################.
+			'isAtomicSite'  => true, // XXX TODO DEBUG DO NOT MERGE.
+			// ################################### WARNING #####################################.
+			'isPrivateSite' => '-1' === get_option( 'blog_public' ),
+		);
+		wp_enqueue_script( 'tiled-gallery-options', '', array(), 'v0.0-inline', true );
```

Comment:
We can get rid of this line by having `wp_add_inline_script` hang from the `jetpack-blocks-editor` handle (like many other Gutenberg-related scripts and oddities in JP).

Reply:
This doesn't work in my testing

Reply:
More accurately, sometimes it works, sometimes it doesn't? Not sure what is going on.

Reply:
Added it here: 86ad09059

Reply:
I'm thinking that some other change (like init -> enqueue_block_assets) allowed this change to work, but it didn't work w/ an earlier version of this branch.